_includes/footer.html

```diff
@@ -55,13 +71,14 @@
     </ul>
     <ul class="menu">
       <li><a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/product/">Product</a></li>
-      <li><a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/confluent-cloud/tryfree/">Confluent Cloud</a></li>
+      <li><a target="_blank" rel="noopener noreferrer"
+          href="https://www.confluent.io/confluent-cloud/tryfree/">Confluent Cloud</a></li>
       <li><a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/ksql/">ksqlDB</a></li>
     </ul>
     <ul class="menu">
       <li><a aria-current="page" class="" href="/">Developer</a></li>
-      <li><a target="_blank" rel="noopener noreferrer" href="https://developer.confluent.io/learn-kafka/">Free Courses</a></li>
```

Comment:
There are a lot of places were `target="_blank"` was removed but `rel="noopener noreferrer"` was kept. However, after removing `target="_blank", there is no longer a need for `rel="noopener noreferrer"`, so both should be removed in places `target="_blank"` was removed.

Reply:
Thanks @iPzard